src/components/index.js

```diff
@@ -1,2 +1,3 @@
 export * from './Button';
```

Comment:
Wydaje mi się, że brakuje componentu GameTypeButton

Reply:
On jest importowany w GameTypes, więc wydaje mi się, że nie jest tu to potrzebne.

Reply:
Nie zauważyłem tego wcześniej. To chyba masz rację :D